qcelemental/models/molecule.py

```diff
@@ -105,26 +110,28 @@ class Molecule(ProtoModel):
         ...,
         # LNN Note: "Ordered" might not be the least ambiguous term
         # (e.g. "ordered" as in linked ordering or "ordered" as in alphabetical?)
-        description="An ordered 1-D array-like object of atomic elemental symbols of shape (N). Index of this list "
+        description="An ordered 1-D array-like object of atomic elemental symbols of shape (NAtom). Index of this list "
```

Comment:
"An ordered `(nat,)` array-like" says the same thing more compactly, if you care for it.

Reply:
I'll take compact, do you have strong opinion on `NAtom` vs `Nat` or `nat` for the shape names? I'm of the opinion that the doc string (and in turn the docs on the page) should be more human-consumable and verbose instead of `nat`, `nfr`, `nvib`, `ndof` which takes some level of contextual or experience to remember what they all mean.

Reply:
I do have strong feelings and hate mixed case, but I won't impose them on docstrings, only the code. :-)

Reply:
I'd rather it be consistent everywhere. I'll go through and lcase them all to match the variables in the code. Hopefully this will be help both end-users and devs.